src/sql/src/plan/error.rs

```diff
@@ -340,6 +341,16 @@ impl PlanError {
             Self::KafkaSourcePurification(e) => e.hint(),
             Self::TestScriptSourcePurification(e) => e.hint(),
             Self::LoadGeneratorSourcePurification(e) => e.hint(),
+            Self::UnknownColumn { table, similar, .. } => {
+                match &similar[..] {
+                    [] => None,
+                    [column] => Some(format!("The similarly named column {} does exist.", ColumnDisplay { table, column })),
+                    names => {
+                        let similar = names.into_iter().map(|column| ColumnDisplay { table, column }).join(", ");
+                        Some(format!("There are similarly named columns that do exist {similar}."))
```

Comment:
```suggestion
                        Some(format!("There are similarly named columns that do exist: {similar}."))
```

(Driveby, feel free to ignore... just reads a bit off to me as-is.)

Reply:
Good thought, updated!